docs/src/components/propsTable/PropsTableExpander.tsx

```diff
@@ -0,0 +1,27 @@
+import { Expander, ExpanderItem } from '@aws-amplify/ui-react';
+import { Properties, Category } from '../../../scripts/types/catalog';
+import { PropsTable } from './PropsTable';
+
+type CategoryProperty = { [key in Category]: Properties };
+type SortedPropertiesByCategory = { [key: string]: Properties }[];
+
+export function PropsTableExpander({
+  propsSortedByCategory,
+}: {
+  propsSortedByCategory: SortedPropertiesByCategory;
+}) {
+  const expanderItem = (categoryProperty: CategoryProperty): JSX.Element => {
+    const title = Object.keys(categoryProperty)[0];
+    return (
+      <ExpanderItem key={title} title={title} value={title}>
+        <PropsTable properties={categoryProperty[title]} />
+      </ExpanderItem>
+    );
+  };
+
+  return (
+    <Expander type="multiple" className="props-table-expander">
```

Comment:
Can we pull in the style changes for the expander from my previous [WIP PR](https://github.com/aws-amplify/amplify-ui/pull/2435/files#diff-cd3edc85d020a29ffeefc5087c3cff0699cc7cd1c701f40c3a8bc3d32dd93930)?  (for .docs-prop-expander). I think we can use a custom style for these single expanders to help them look more clickable and related to the table above it. See example of what they should look like here: https://docs-prop-tables-refactor.dvmvffzts1tcu.amplifyapp.com/react/components/table?tab=props

Note: Just realized that style^ is missing a focus style though so we should add that

Reply:
Thanks, Heather! The expander looks a lot better 🎉 

Yea, the focus style is done already in this PR https://refactor-props-table.dy9yokhh4908z.amplifyapp.com/react/components/table?tab=props

(the latest commit is still deploying...)